bin/docs.test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -rf loopback.io
+git clone https://github.com/strongloop/loopback.io.git
+npm run bootstrap
```

Comment:
You should be able to clone `loopback.io` into `sandbox` instead of creating its own location.

Reply:
The other option is to have `loopback.io` repo checked out somewhere else and try the following:

```
cd loopback.io
npm i ~/Projects/loopback-next/docs --no-save
```

Reply:
Good call for using sandbox. 

---

We would still need to run the other commands (fetch-readmes, bundle install, jekyll build) since it's the build step that verifies the site hasn't broken due to formatting errors in docs, missing files, etc.

Reply:
Something like this? https://github.com/strongloop/loopback.io/pull/666